include/drivers/sensor.h

```diff
@@ -177,6 +177,9 @@ enum sensor_channel {
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
 
+	/** Fingerprint Enrolling and searching channel */
```

Comment:
units?

Reply:
It is fingerprint sensor, it will give template(fingerprint) counts and we can pass ID number for storing fingerprint in the sensor . I hope latest modified comment is enough.